From edc5b114a098df5fa50f92882073c73c34bedaf2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 29 Nov 2016 17:34:29 +0100
Subject: [PATCH] Do not use done_testing with Test::NoWarnings
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Test::NoWarnings does not work with done_testing correctly and the
mistaken usage in t/09warnings.t becomes visible since
Test-Simple-1.302064.

This patch replaces done_testing call with proper test plan
definition.

CPAN RT#119030

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 t/09warnings.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/09warnings.t b/t/09warnings.t
index 3ee58eb..114b5a6 100644
--- a/t/09warnings.t
+++ b/t/09warnings.t
@@ -27,7 +27,7 @@ the same terms as the Perl 5 programming language system itself.
 
 use strict;
 use warnings;
-use Test::More;
+use Test::More tests => 1;
 use Test::Requires "List::MoreUtils";
 
 BEGIN {
@@ -40,5 +40,3 @@ use Test::NoWarnings;
 $^W = 1;
 require Scalar::Does;
 Scalar::Does::does(undef, 'ARRAY');
-
-done_testing(1);
-- 
2.7.4

